### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@bersace opm ?

File: hookforward/uninstall.sls [hunk 1]
```diff
@@ -4,26 +4,4 @@ include:
...
-  pkg.purged:
-    - pkgs:
-      - nodejs
-
-remove_node_link:
-  file.absent:
-    - name: /usr/bin/node
-    - require:
-      - pkg: remove_nodejs
+  npm.removed
```
Comment (bersace) [inline hookforward/uninstall.sls]: Missing `- name: hookforward`.

File: jenkins/cli/init.sls [hunk 2]
```diff
@@ -11,11 +11,20 @@ curl_pkg:
   pkg.latest:
     - name: curl
 
+{%- if 'jenkins-master' in grains['roles'] %}
```
Comment (bersace) [inline jenkins/cli/init.sls]: Put hyphen at end of if

File: jenkins/master/init.sls [hunk 3]
```diff
@@ -37,7 +36,7 @@ jenkins_config:
 
 ssh_key:
   cmd.run:
-    - name: ssh-keygen -q -N '' -f {{ home }}/.ssh/id_rsa
+    - name: test -f  {{ home }}/.ssh/id_rsa || ssh-keygen -q -N '' -f {{ home }}/.ssh/id_rsa
     - user: {{ user }}
     - creates: {{ home }}/.ssh/id_rsa
```
Comment (bersace) [inline jenkins/master/init.sls:41]: Redundant. Je check.

**Pull request merged**